client/tests/e2e/position.js

```diff
@@ -375,6 +375,10 @@ test.serial("Update permissions while changing positions", async t => {
   await navigateOrganization("EF 5")
   await navigateSubOrganization("EF 5.1")
 
+  // Refresh organization page.
+  await t.context.driver.navigate().refresh()
```

Comment:
Do you really need to refresh the whole page here? It will mean a full reload of the React app. Isn't it sufficient just to wait for the SVG to be redrawn by the D3 code?